src/main/groovy/wooga/gradle/xcodebuild/traits/XcodeArchiveActionSpec.groovy

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2018-2020 Wooga GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ *
+ */
+
+package wooga.gradle.xcodebuild.traits
```

Comment:
Can we please don't do this. A trait is a trait. We don't create packages with `classes` or `interfaces`. The traits belong to the public interface and if you can't group them any further just leave them in the base package `wooga.gradle.xcodebuild`

Reply:
Would the term `models` work for you?